client/boot/project/wordpress-com.js

```diff
@@ -255,7 +239,7 @@ function reduxStoreReady( currentUser, reduxStore ) {
 	}
 
 	if ( config.isEnabled( 'keyboard-shortcuts' ) ) {
-		require( 'lib/keyboard-shortcuts/global' )( sites ); // GENERIC? WHY SITES
+		require( 'lib/keyboard-shortcuts/global' )( sitesFactory() );
```

Comment:
It isn't enabled on production but it works in other envs. I think we can try to pass redux store instead and see if it works :)